### DISCUSSION TITLE
Mini HUD andere Version

### DISCUSSION THREAD

**Discussion by Hobbylohner** [Courseplay/Courseplay_FS22#3245] [Q&A]
Hallo, ich habe eine Frage.
und zwar habe ich die aktuellste Version 7.4.1.2 heruntergeladen. 
Allerdings erhalte ich immer nur das Mini GUI auf dem 2. Bild möchte allerdings den HUD auf dem ersten Bild nutzen, da mir die Nutzung damit deutlich leichter fällt. Durch klicken mit beiden Maustasten passiert nichts und mit der Löschen Taste wird nur der Mini GUI geöffnet. 
Habe schon alle Tastenbelegungen durchprobiert aber der HUD wird mir nicht angezeigt.
Würde mich über eine Antwort freuen.
LG
Bild 1.
![image](https://github.com/Courseplay/Courseplay_FS22/assets/170087602/88c5411e-545e-4cd0-a84a-b3044aeacd91)
Bild 2.
![image](https://github.com/Courseplay/Courseplay_FS22/assets/170087602/fda151f0-28f3-4820-86f3-4cb59f42b3d6)

**Comment by Tensuko** [comment]
Irgendwie verwechselst du hier einiges....
Das mini-hud ist das erste bild, rechte maustaste oder entf (tastenbelegung mal überprüft? kommt ein mauszeiger?)

Das zweite bild ist das normale AI Menü.

**Comment by rohne83** [comment]
Tastaturbelegungen sollte man wohl in jedem Fall überprüfen. Möglicherweise einfach keine Tastaturbelegung (bzw in diesem Fall eher Maus) zum Öffnen des Huds mehr eingestellt, oder was auch Probleme machen kann dieselbe schon für was anderes verwendet.
